packages/grafana-ui/src/components/Graph/Graph.tsx

```diff
@@ -1,10 +1,9 @@
 // Libraries
 import $ from 'jquery';
 import React, { PureComponent } from 'react';
-import uniqBy from 'lodash/uniqBy';
+import _, { uniqBy } from 'lodash';
```

Comment:
I'm not sure why it's not a lint error, but I don't think the default `_` export is even used in here?